package.json

```diff
@@ -19,13 +20,14 @@
   },
   "homepage": "https://github.com/LateralView/spotify-exercise#readme",
   "dependencies": {
-    "angular": "^1.5.8",
-    "angular-ui-router": "^0.3.2",
+    "angular": "^1.6.0",
+    "angular-ui-router": "^1.0.0-beta.3",
```

Comment:
Yes, on our project we plan to move to 1.6 but we currently have modules that doesn't support that, also, the error is fine because warns you that you aren't handling all the promises. But is benevolent and doen't cause any side effect.

Also, the beta of ui-router is pretty nice, but they deprecate the $stateChange events to let plate to his "transforms" i find that pretty nice and performant but breaks a lot of applications.